src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -882,4 +882,24 @@ object SampleWdl {
     val OutputKey1 = "hello.msg1.result"
     val OutputValue = "waited 100 seconds"
   }
+
+  object BadTaskOutputWdl extends SampleWdl {
+    override def wdlSource(runtime: String): WdlSource =
+      """task bad {
+        |	command {
+        |		echo "hello" > a
+        |	}
+        |	output {
+        |		# Oops! we made a spelling mistake in our WDL!
+        |		File a = "b"
+        |	}
+        |}
+        |
+        |workflow badExample {
+        |	call bad
+        |}
```

Comment:
Do we have WDL coding standards?  :smile:  I'd advocate for less than 8 chars of indent per level.